styleguide/source/_patterns/03-organisms/tools.json

```diff
@@ -1,9 +1,9 @@
 {
   "tools": {
     "heading": {
-      "class": "ama__h5",
+      "class": "ama__h4",
```

Comment:
Should we be adding the classes in the twig instead of the json?

Reply:
I'm not sure what we decided on.

Reply:
@labbydev Neil thinks that it doesn't matter.

Reply:
@nstriedinger @labbydev My opinion is that we should be adding them in the Twig whenever possible.

Reply:
According to FED meeting notes from 3/20/18
> Headings will continue to leave the class in the Twig